src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -440,3 +440,38 @@
         visibility: hidden;
     }
 }
+
+/***** Error Toolbar *****/
+.sfErrorToolbar {
```

Comment:
btw, not having the `sf-toolbar` class means you also need to handle the case of the error toolbar separately in the print version.